src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+
+## Summary
+
+Participating agencies/organizations use GSA SmartPay charge cards to obtain approximately $30 billion in products and services annually in support of their missions through approximately 100 million purchase, travel and fleet transactions. The CHARGE Act emphasizes the existing ability for agencies/organizations to use GSA SmartPay charge cards to pay for charging services for EVs at commercial charging facilities. At the time of this guidance being issued, GSA SmartPay contractor bank charge cards, including WEX and Voyager fleet cards, can be used to pay for charging services at ChargePoint, Inc. commercial charging stations. Other charging station networks currently only accept Visa & MasterCard branded cards through their respective payment apps and terminals. Certain EVgo chargers do include card readers for walk-up payment, but not all. As the electric vehicle charging industry continues to expand, the Center for Charge Card Management (CCCM) and our GSA SmartPay contractor banks will pursue additional relationships with charging station providers to increase access and acceptance at additional charging networks for federal agencies. Use of GSA SmartPay charge cards for both per transaction and charging subscriptions is permissible consistent with agency policy, applicable charge card spending limit(s), the Federal Acquisition Regulation and other applicable rules/regulations. 
+
+## Action
+
+Each Federal agency may, for Government owned or managed EVs used in accordance with 31 U.S.C 1344, “Passenger Carrier Use,” (1) allow for the vehicles to be charged at a commercial EV charging station and (2) pay for that charging with a GSA SmartPay charge card. The type of GSA SmartPay charge card to be used (purchase, travel, integrated and/or fleet) to complete the charging transaction must comply with agency policy. GSA SmartPay charge cards may also be used to pay for commercial rental car EV changing when an employee is in official travel status. Additional operational considerations for use of charge cards pursuant to the CHARGE Act are provided below:
+
+### GSA Leased Vehicles
+-  To enable the WEX fleet card for use at ChargePoint stations, the following options are available:
```

Comment:
To enable the WEX fleet card for use at ChargePoint stations, the following options are available:
For agencies with at least one owned ChargePoint station, WEX accounts can be linked to ChargePoint RFIDs in the data portal or through the ChargePoint website. This action must be taken by the agency's fleet manager (s).
For agencies/drivers that want to charge at public ChargePoint stations, contact GSA Fleet’s Alternative Fuel Vehicle Team at gsafleetafvteam@gsa.gov and provide EV tag number(s) and delivery location(s). GSA Fleet will provide the driver with ChargePoint Radio Frequency IDentification (RFID) card(s) for the vehicle(s) linked to the vehicle’s associated GSA SmartPay fleet charge card. GSA Fleet EV commercial charging information resources include charging station purchasing options and charging procedures at [gsa.gov/evse](http://gsa.gov/evse).
If an agency desires to use additional commercial charging station vendors beyond ChargePoint for GSA Fleet leased EVs, it must first contact the GSA Fleet Alternative Fuel Vehicle Team at gsafleetafvteam@gsa.gov.